modules/caddyhttp/reverseproxy/caddyfile.go

```diff
@@ -174,6 +177,12 @@ func (h *Handler) UnmarshalCaddyfile(d *caddyfile.Dispenser) error {
 		if network != "" {
 			return caddy.JoinNetworkAddress(network, host, port), nil
 		}
+		// if both network and port are empty, host address contains
+		// placeholdler (e.g. "{http.request.header.X-Caddy-Upstream-Dial}") and doing net.JoinHostPort(host, port) will
```

Comment:
```suggestion
		// placeholder (e.g. "{http.request.header.X-Caddy-Upstream-Dial}") and doing net.JoinHostPort(host, port) will
```

Reply:
I'm surprised golangci-lint didn't catch it 🤨